src/flows/data_retention/user_snowplow_events_flow.py

```diff
@@ -0,0 +1,31 @@
+"""Delete all raw snowplow events for users who have requested to have their accounts deleted.
+
+ Reference: https://getpocket.atlassian.net/wiki/spaces/CP/pages/2703949851/Snowflake+Data+Deletion+and+Retention
+"""
+
+from prefect import Flow
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+from prefect.tasks.dbt import DbtCloudRunJob
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
+from utils.flow import get_flow_name
+
+FLOW_NAME = get_flow_name(__file__)
+
+# Run first day of month at midnight UTC.
+SCHEDULE = Schedule(clocks=[CronClock("0 0 1 * *")])
+
+DBT_MODEL_CLEANUP_JOB_ID = 17774
+
+DELETE_SQL = """
+DELETE FROM "SNOWPLOW"."ATOMIC"."EVENTS" as e
+USING "ANALYTICS"."DBT_STAGING"."STG_ACCOUNT_DELETIONS" as d 
+WHERE e.CONTEXTS_COM_POCKET_USER_1[0]:hashed_user_id = d.hashed_user_id
+"""
+
+with Flow(FLOW_NAME, schedule=SCHEDULE) as flow:
+    snowflake = PocketSnowflakeQuery()
```

Comment:
@jeshuaborges Would it be possible to create a special Snowflake user/connection for this, so that all our Prefect development doesn't happen with a connection that can delete from one of our most important tables? Or some other mitigation.

Reply:
@kennylong great suggestion. I was also thinking that this would be a good idea. We can absolutely do this.